manifests/p/Prometheus/WindowsExporter/0.30.0/Prometheus.WindowsExporter.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with komac v2.9.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Prometheus.WindowsExporter
+PackageVersion: 0.30.0
+PackageLocale: en-US
+Publisher: Prometheus Community
+PublisherUrl: https://github.com/prometheus-community
+PublisherSupportUrl: https://github.com/prometheus-community/windows_exporter/issues
+PackageName: windows_exporter
+PackageUrl: https://github.com/prometheus-community/windows_exporter
+License: MIT
+LicenseUrl: https://github.com/prometheus-community/windows_exporter/blob/master/LICENSE
+Copyright: Copyright (c) 2016 Martin Lindhe
+CopyrightUrl: https://github.com/prometheus-community/windows_exporter/blob/master/LICENSE
+ShortDescription: Prometheus exporter for Windows machines
+Description: A Prometheus exporter for Windows machines.
+Moniker: windows_exporter
+Tags:
+- hacktoberfest
```

Comment:
The tag `hacktoberfest` is reference in the source code [context](https://hacktoberfest.com/), but does not make sense having this tag on a package manager.